test/functional/database-schema/column-types/cockroachdb/entity/Post.ts

```diff
@@ -142,6 +142,56 @@ export class Post {
     @Column("inet")
     inet: string
 
+    // -------------------------------------------------------------------------
+    // Geometry Type
+    // -------------------------------------------------------------------------
+
+    @Column("geometry")
+    point: Object
```

Comment:
@AlexMesser what is the actual type for the "Object"?

Reply:
GeoJSON/Point I believe